apps/roles_test.go

```diff
@@ -0,0 +1,73 @@
+package apps
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRoleHandlers(t *testing.T) {
```

Comment:
merge these into one test with subtests and without calling assert.New()

Reply:
done